CHANGELOG.md

```diff
@@ -1,5 +1,50 @@
 # ChangeLog
 
+## 29 December 2016, build 158
+
+I forgot to update this in a while, so this is a quite big changelog entry.
+
+I got the first major code contributions by other developers, so from now on the
+changelog entries will be split by author. It may also have a header like this
+one in which it will contain a short summary or a message from the author.
+
+Special thanks to Hugo Rosnet, who contributed a lot of code that implemented a
```

Comment:
I'd prefer my github nickname rather than explicit first/last name, despite being in my commit messages.